test/components/structures/MatrixChat-test.tsx

```diff
@@ -1104,7 +1079,7 @@ describe("<MatrixChat />", () => {
                         homeserverUrl +
                         " softLogout: " +
                         false,
-                    " freshLogin: " + false,
+                    " freshLogin: " + true,
```

Comment:
What changed?

Reply:
fair question. I haven't really dug into it, because intuitively, I'd expect this to log `true` - it being a fresh login and all.

I think this is happening because we're now using proper local- and session-storage rather than the crappy mocks we had before.